### ISSUE TITLE
Too many orphan objects for `git gc`

### ISSUE THREAD

**Issue by mguaypaq** [neuropoly/data-management#276]
When I pushed `master` at the end of #275, I got a warning message from the server:
```
remote: Résolution des deltas: 100% (57/57), complété avec 6 objets locaux.
remote: warning: Le dernier lancement de gc a rapporté l'erreur suivante. Veuillez corriger
remote: la cause et supprimer gc.log
remote: Le nettoyage automatique n'aura pas lieu jusqu'à ce que le fichier soit supprimé.
remote: 
remote: warning: Il y a trop d'objets seuls inaccessibles ; lancez 'git prune' pour les supprimer.
remote: 
```
I'm not quite sure what it means, but I should investigate. In any case, the push worked, and the message says that git won't do anything dangerous until we tell it to, so I'm not too worried.

**Comment by mguaypaq** [comment]
Probably related to #271.

**Comment by mguaypaq** [comment]
I ran `git fsck` to get a (very large) list of unreachable objects. Most of them were blobs or trees, but there were about a dozen unreachable commit objects, too. For the commits, I checked that they were either:
* old git-annex commits that recently got cleaned up; or
* commits that likely got rebased, because the master branch contains a commit with the same message and timestamp.

So, it looks like all of the unreachable commits really are useless. I ran `git prune` to get rid of them, and removed the `gc.log` file as instructed in the original warning message.
